<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignTransformer</class>
 <widget class="QDialog" name="SignTransformer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>928</width>
    <height>455</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SignTransformer</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pushButton_out_dir">
         <property name="text">
          <string>Config Outdir</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_outdir"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QListView" name="listView_ins"/>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="pushButton_open_signs">
           <property name="text">
            <string>Open Signs</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QGraphicsView" name="graphicsView_ws"/>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_tx_signs">
           <property name="text">
            <string>Transform Signs</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QListView" name="listView_outs"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
